src/components/AccountDetails.js

```diff
@@ -13,9 +26,19 @@ export default class AccountDetails extends Component {
     onDeleteAccountPressed: PropTypes.func.isRequired,
   }
 
+  constructor(props) {
+    super(props)
+    this.state = {}
```

Comment:
This should be part of the class:
```
class AccountDetails ... {
 state = {};
}
```

and `displayIcon` should be called in `componentDidMount`
```
componentDidMount () {
  displayIcon(this, `0x` + this.props.account.address)
}
```
TBH `displayIcon` could be just part of the class and if we should also call this method on `componentWillReceiveProps`, but currently I guess it's not possible for a parent to modify `this.props.account` so it should work fine.